tests/Unit/Domain/Test_CoordinatesTag.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/DataBox.hpp"
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/Tags.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+template <size_t Dim, typename T>
+void test_coordinates_compute_item(Index<Dim> extents, T map) noexcept {
+  using map_tag = Tags::ElementMap<Dim>;
+  const auto box = db::create<
+      db::AddTags<Tags::Extents<Dim>, map_tag>,
+      db::AddComputeItemsTags<
+          Tags::LogicalCoordinates<Dim>,
+          Tags::Coordinates<map_tag, Tags::LogicalCoordinates<Dim>>>>(
+      extents, ElementMap<Dim, Frame::Grid>(
+                   ElementId<Dim>(0),
+                   make_coordinate_map_base<Frame::Logical, Frame::Grid>(map)));
+  CHECK_ITERABLE_APPROX(
+      (db::get<Tags::Coordinates<map_tag, Tags::LogicalCoordinates<Dim>>>(box)),
```

Comment:
will I have to fetch the coordinates this way?  I would like to get them with something like db::get<Tags::Coordinates<Frame>>(box)

Reply:
Yes, currently the long way is necessary. We could alternatively have things like `Tags::InertialCoordinates` etc. Another possibility might be to just template on the target frame. This would require making a canonical decision on source frame, and map. That could be a bit tricky with ElementMap and dual frames. I'd have to think about it.